<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managebooks</class>
 <widget class="QDialog" name="managebooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>440</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>迷你简稚艺</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除所选</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>821</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>迷你简稚艺</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>8</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>检索</string>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>按书名</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISBN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>作者</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>出版社</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>440</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>迷你简稚艺</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认修改</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>1351</width>
     <height>343</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="rowCount">
    <number>20</number>
   </property>
   <property name="columnCount">
    <number>8</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>书名</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ISBN</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>作者</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>出版社</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>馆藏数量</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>可借数量</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>关键字</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>简介</string>
    </property>
    <property name="font">
     <font>
      <family>迷你简稚艺</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>275</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_6</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
